packages/validation-manager/src/ValidationManager.ts

```diff
@@ -118,7 +121,11 @@ export class ValidationManager implements IValidationManager {
       factory != null && factory !== EIP_7702_MARKER_INIT_CODE ? this.entryPoint.getDepositInfo(factory) : null
     ])
     return {
-      sender: { addr: sender, stake: senderInfo.stake, unstakeDelaySec: senderInfo.unstakeDelaySec },
+      sender: {
```

Comment:
If you have time, please undo the formatting changes for cleaner diff

Reply:
will undo. it was automatic. better we share the same prettier config for idea